### PR TITLE
add config from previous PR

### PR DESCRIPTION
**Pull request by mqhackett**
Signed-off-by: mqhackett <mahacket@redhat.com>

### CODE REVIEW

File: ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
```
Comment (jlpadilla) [inline ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml:9]: Let's use go 1.17.  We upgraded all our projects recently.

File: ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- as: test-unit
+  commands: |
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow unit-test
+  container:
+    from: src
+- as: sonar-pre-submit
+  commands: |
+    export SELF="make -f Makefile.prow"
+    export DEPLOYED_IN_HUB=true
```
Comment (jlpadilla) [inline ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml:60]: Remove this line, this env is specific for our collector and has no effect here.
```suggestion
```

File: ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    make -f Makefile.prow sonar/go/prow
+  container:
+    from: src
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
+- as: sonar-post-submit
+  commands: |
+    export SELF="make -f Makefile.prow"
+    export DEPLOYED_IN_HUB=true
```
Comment (jlpadilla) [inline ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml:70]: Remove from here too (see comment above).
```suggestion
```

File: ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-release-2.5.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: ocm-ci-image-mirror
+- as: sonar-post-submit
+  commands: |
+    export SELF="make -f Makefile.prow"
+    export DEPLOYED_IN_HUB=true
```
Comment (jlpadilla) [inline ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-release-2.5.yaml:75]: Remove this line (see previous comment)
```suggestion
```

File: ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
```
Comment (jlpadilla) [inline ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml:13]: Update namespace.
```suggestion
    namespace: stolostron
```

File: ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
+  to: search-indexer
+promotion:
+  name: "2.5"
+  namespace: open-cluster-management
```
Comment (jlpadilla) [inline ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml:25]: Update namespace
```suggestion
  namespace: stolostron
```

File: ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: search-indexer
+    env:
+      IMAGE_REPO: search-indexer
+      REGISTRY_ORG: open-cluster-management
```
Comment (jlpadilla) [inline ci-operator/config/config/stolostron/config-stolostron-open-cluster-management-search-indexer-main.yaml:38]:
```suggestion
      REGISTRY_ORG: stolostron
```

**Comment by mqhackett** [comment]
Closing this PR as the changes will be made in https://github.com/openshift/release/pull/25257/

**Pull request closed without merging**